project/plugins.sbt

```diff
@@ -1,20 +1,17 @@
-resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
+resolvers += Resolver.typesafeRepo("releases")
 
-resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.6.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.15.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.4.0")
```

Comment:
The git-versioning dependency can be removed now as it's brought in as a transitive dependency by sbt-auto-build since v 3.0.0 https://confluence.tools.tax.service.gov.uk/display/TEC/2021/05/05/New+version+of+sbt-auto-build